project/src/main/java/com/google/autograder/servlets/GetAssignmentDetailsServlet.java

```diff
@@ -27,8 +28,8 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         Filter courseIDFilter = new FilterPredicate("courseId", FilterOperator.EQUAL, courseID);
 
         Query query = new Query("Assignment").setFilter(courseIDFilter).setFilter(assignmentIDFilter);
```

Comment:
Another method that would go well in the Database class is `getAssignment(courseId, assignmentId)`, then callers won't have to care about constructing queries. Ideally, callers would not need to import anything starting with `com.google.appengine.api.datastore`, leaving all the details of Datastore to the Database class. This can be in a future PR.

Reply:
Ok, definitely. I also noticed that with implementing your above comments in the Database class really only about four methods end up dealing directly with the DATA_STORE variable. It would make sense to do the same with calls outside the class as well.